flask/boxwise_flask/graph_ql/query_defs.py

```diff
@@ -10,9 +10,14 @@
         base(id: Int!): Base
         allUsers: [User]
         user(email: String): User
-        box(qr_code: String): Box
+        box(id: String): Box
+        location(id: String): Location
+        locations: [Location]
+        qrCode(id: Int!): QrCode
         qrExists(qr_code: String): Boolean
-        qrBoxExists(qr_code: String): Boolean
+        boxIdByQrCode(qr_code: String): Int
```

Comment:
I'd remove this query since one can write ```
query { qrCode (id: ID) { box { box_id } } }
```
instead

Reply:
Yes, that would be perfect - but is this already implemented on master in the backend? And I think instead of id we need to pass in the qr_code to the qrCode() call, right?

Reply:
It's part of the #59 PR, and about to come to master. Yep, has to be qr_code as an argument!

Reply:
FYI the latest commit removes the boxIdByQrCode query.